tests/testsuite/package.rs

```diff
@@ -3304,3 +3304,57 @@ fn init_and_add_inner_target(p: ProjectBuilder) -> ProjectBuilder {
     .file("derp/target/foo.txt", "")
     .file("derp/not_target/foo.txt", "")
 }
+
+#[cargo_test]
+fn custom_build_warning() {
```

Comment:
nitpick: this test name could be better to tell what it tests against, something like `build_script_outside_pkg_root`?